### ISSUE TITLE
[Bug] 0-welcome.yml doesn't run

### ISSUE THREAD

**Issue by Maxim-Mazurok** [skills/write-javascript-actions#44]
`0-welcome.yml` workflow didn't run on repo creation, had to run it manually

**Comment by hectorsector** [comment]
Thanks for the report, @Maxim-Mazurok. Duplicate of https://github.com/skills/write-javascript-actions/issues/37.

**Comment by Maxim-Mazurok** [comment]
@hectorsector I wasn't sure if it is a duplicate, because the first workflow is triggered on repo creation. The other issue is about not triggering on PR I believe.

**Comment by sinsukehlab** [comment]
I agree with @Maxim-Mazurok. This issue is about the Step 0 workflow, and #37 is about the Step 1 workflow.
I think this issue is related to skills/code-with-codespaces#87 and skills/copilot-codespaces-vscode#136.

**Comment by hectorsector** [comment]
Apologies! Thanks for catching that @Maxim-Mazurok @sinsukehlab
